tools/ci_build/github/azure-pipelines/templates/win-ci-vs-2019.yml

```diff
@@ -181,7 +181,7 @@ jobs:
       solution: '$(Build.SourcesDirectory)\csharp\OnnxRuntime.CSharp.sln'
       configuration: '${{ parameters.BuildConfig }}'
       platform: 'Any CPU'
-      msbuildArguments: '-p:OnnxRuntimeBuildDirectory="$(Build.BinariesDirectory)" -p:OrtPackageId=$(OrtPackageId) -p:DefineConstants=USE_${{ parameters.ORT_EP_NAME }}'
```

Comment:
If you remove this part, The EP specific tests will not run.

Reply:
If you use `--build_csharp` (we do on line 145), I believe onnxruntime_csharp.cmake will update csharp/Directory.Build.props, which sets which EPs are enabled via  'USE_xxx' defines.  

This is a better setup as it a) allows an arbitrary combination of EPs to be used and b) doesn't use a `/p` property which is a global value that overrides any values set in csproj files.

Reply:
Got it. Thank you!